lib/ui/semantics.dart

```diff
@@ -547,8 +559,20 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     Float64List transform,
     Int32List childrenInTraversalOrder,
     Int32List childrenInHitTestOrder,
+    Int32List customAcccessibilityActions,
   ) native 'SemanticsUpdateBuilder_updateNode';
 
+  /// Update the custom accessibility action associated with the given `id`.
+  /// 
+  /// The name of the action exposed to the user is the `label`. The text 
+  /// direction of this label is the same as the global window.
+  void updateAction({int id, String label}) {
```

Comment:
Should this be updateCustomAction? Or can you also update a non-custom action with a label? What would that mean?

Reply:
Changed to updateCustomAction